workflows/dags/data/bulk_data_harvest.py

```diff
@@ -210,12 +212,11 @@ def normalize_collaborations(records):
         Returns: dict: The normalized record.
         """
         normalized_records = []
+
+        logger.info(f"Normalizing {len(records)} records")
         for record in records:
             collaborations = record.get("collaborations", [])
 
-            if not collaborations:
-                continue
-
             response = inspire_http_hook.call_api(
```

Comment:
If you would like to avoid making extra calls is missing collaborations you can wrpa this part to an
`if collaborations:`

If we have collaborations, we do the normalization, if we do not have we do nothing.

Either way we append the record.